classic/form/field/GeocoderComboBox.js

```diff
@@ -79,6 +79,14 @@ Ext.define('GeoExt.form.field.GeocoderComboBox', {
      */
     store: null,
 
+    /**
+     * The property in the JSON response of the geocoding service used in
+     * the store's proxy as root object.
+     *
+     * @cfg {String}
+     */
+    proxyRootProperty: null,
```

Comment:
this is a required parameter, right? if so, can we mark it as required in the API doc?

Reply:
No it's not mandatory. If not given the root array will be taken (see [here](https://docs.sencha.com/extjs/6.2.0/classic/Ext.data.reader.Json.html#cfg-rootProperty) for more details).